test/PushNotificationTests/UnpackagedSelfContained.cpp

```diff
@@ -0,0 +1,214 @@
+            const PACKAGE_VERSION c_minVersion{};
+            VERIFY_ARE_EQUAL(S_OK, MddBootstrapInitialize(c_Version_MajorMinor, nullptr, c_minVersion));
+
+            return true;
+        }
+
+        TEST_METHOD_CLEANUP(MethodUninit)
+        {
+            // Need to keep each TEST_METHOD in a clean state in the LRP.
+            try
+            {
+                PushNotificationManager::Default().UnregisterAll();
+            }
+            catch (...)
+            {
+                // We want to unregister regardless of pass/fail to clean the state.
+            }
+            MddBootstrapShutdown();
+
+            VERIFY_IS_TRUE(TP::IsPackageRegistered_WindowsAppRuntimeFramework());
+            VERIFY_IS_TRUE(TP::IsPackageRegistered_DynamicDependencyDataStore());
+            VERIFY_IS_TRUE(TP::IsPackageRegistered_DynamicDependencyLifetimeManager());
+            VERIFY_IS_TRUE(TP::IsPackageRegistered_PushNotificationsLongRunningTask());
+
+            return true;
+        }
+
+        HRESULT ChannelRequestHelper(IAsyncOperationWithProgress<PushNotificationCreateChannelResult, PushNotificationCreateChannelStatus> const& channelOperation)
+        {
+            if (channelOperation.wait_for(c_timeout) != winrt::Windows::Foundation::AsyncStatus::Completed)
+            {
+                channelOperation.Cancel();
+                return HRESULT_FROM_WIN32(ERROR_TIMEOUT); // timed out or failed
+            }
+
+            auto result{ channelOperation.GetResults() };
+            auto status{ result.Status() };
+            if (status != PushNotificationChannelStatus::CompletedSuccess)
+            {
+                return result.ExtendedError(); // did not produce a channel
+            }
+
+            result.Channel().Close();
+            return S_OK;
+        }
+
+        TEST_METHOD(ChannelRequestUsingNullRemoteId)
+        {
+            auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(winrt::guid()) };
+            VERIFY_ARE_EQUAL(ChannelRequestHelper(channelOperation), E_FAIL);
+        }
+
+        TEST_METHOD(ChannelRequestUsingRemoteId)
+        {
+            auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(c_azureRemoteId) };
+            VERIFY_ARE_EQUAL(ChannelRequestHelper(channelOperation), E_FAIL);
+        }
+
+        // Currently failing.
+        TEST_METHOD(MultipleChannelClose)
+        {
+            PushNotificationTestFunctions::MultipleChannelClose();
+        }
+
+        TEST_METHOD(VerifyRegisterAndUnregister)
+        {
+            PushNotificationTestFunctions::VerifyRegisterAndUnregister();
+        }
+
+        TEST_METHOD(VerifyRegisterAndUnregisterAll)
+        {
+            PushNotificationTestFunctions::VerifyRegisterAndUnregisterAll();
+        }
+
+        TEST_METHOD(MultipleRegister)
+        {
+            auto pushNotificationManager{ PushNotificationManager::Default() };
```

Comment:
Lets remove the local initialization and directly infer the pushManager via Default